src/python/pants/backend/python/dependency_inference/module_mapper_test.py

```diff
@@ -61,13 +62,19 @@ def test_default_module_mapping_is_normalized() -> None:
 
 
 def test_default_module_mapping_uses_tuples() -> None:
-    for modules in DEFAULT_MODULE_MAPPING.values():
+    for modules in [
+        *DEFAULT_MODULE_MAPPING.values(),
+        *DEFAULT_TYPE_STUB_MODULE_MAPPING.values(),
+    ]:
         assert isinstance(modules, tuple)
         assert len(modules) > 0
 
 
 def test_default_module_pattern_mapping_keys_and_value_types() -> None:
-    for pattern, replacements in DEFAULT_MODULE_PATTERN_MAPPING.items():
+    for pattern, replacements in [
+        *DEFAULT_MODULE_PATTERN_MAPPING.items(),
+        *DEFAULT_TYPE_STUB_MODULE_PATTERN_MAPPING.items(),
+    ]:
```

Comment:
this and the above add symmetry between the two. From here till the last comment in this file are just `pants fmt` changes